src/components/Header.js

```diff
@@ -1,10 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
useState선언은 왜 해주신걸까요 ? 부모컴포넌트에서 props로 받아서 바로 사용하기 때문에  Header에서는 useState를 또 선언해주실필요가 없답니당

```suggestion
import React from 'react';
```